tox.ini

```diff
@@ -3,8 +3,14 @@
 envlist = py27
 
 [testenv]
-deps = -r{toxinidir}/requirements/test.txt
+deps = 
+	-r {toxinidir}/requirements/test.txt
+	-e git+https://github.com/edx/xblock-sdk.git@21dac6a88fad2cf6d5588b0ee5a1e7c7d3b591fc#egg=xblock-sdk==0.1.6
```

Comment:
This is not needed here, it is already in test.txt

Reply:
Agreed. This line slipped by during tests.